imqsauth/http.go

```diff
@@ -154,6 +159,10 @@ func (x *ImqsCentral) RunHttp() error {
 	smux.HandleFunc("/users", makehandler(HttpMethodGet, httpHandlerGetUsers, 0))
 	smux.HandleFunc("/groups", makehandler(HttpMethodGet, httpHandlerGetGroups, 0))
 
+	smux.Handle("/authnotifications", websocket.Handler(x.handleWebSocket))
```

Comment:
Does the word "auth" belong in this API, when this is already inside the "auth" service?